resume.yaml

```diff
@@ -1,121 +0,0 @@
-name: Sean Gatewood
-contact:
-  email: stg2bd@virginia.edu
-  linkedin: linkedin.com/in/sean-t-gatewood
-  github: github.com/sgatewood
-education:
-  school: University of Virginia
-  school titles:
-    - Rodman Scholar
-    - Raven Society
-  degree: B.S. Computer Science
-  gpa: 3.95
-  graduation: May 2020
```

Comment:
I didn’t do this on mine, but I’ve heard that taking the graduation year off of your resume might improve your chances at being considered for a more senior role (obviously assuming you prove your skills in the interview; but the video said it helps get past that initial screening). Might be worth looking into